src/components/Table/nestedCards.tsx

```diff
@@ -189,6 +189,28 @@ export function maybeAddCardPadding(openCards: NestedCardStyle[], column: "first
   );
 }
 
+/**
+ * Wraps a row within its parent cards. Creates a wrapping div to add the card padding.
+ * Example:
+ * <div parent> <div child> <div grandchild /> </div> </div>
+ */
+export function maybeWrapCard(openCards: NestedCardStyle[], row: ReactNode): any {
```

Comment:
Maybe call this `maybeWrapLeafCard`? And maybe call the `row` param `leaf`? 

Or can we remove the `maybe`, and just call it `wrapLeafCard`? From looking at the call site, I think it already only calls this method if we know there are open cards.

Reply:
I started out with that naming, but this isn't just used for leaf rows. This function is called for all GridRows where `openCardStyles !== undefined`. Though, probably could call this only when `openCardStyles !== undefined && openCardStyles.length > 0`. 
If we ever need to "add card padding" then this is called. In the example of `Parent -> Child -> GrandChild`, then this wrapping function should be executed for both `Child` and `GrandChild` to make sure the GridRow element is contained within each `openCardStyles`.

Reply:
Ah right right, that makes sense.

Reply:
I'll update that conditional for calling this function and then rename it to just `wrapCard`.

Reply:
> In the example of Parent -> Child -> GrandChild, then this wrapping function should be
> executed for both Child and GrandChild to make sure the GridRow element is contained
> within each openCardStyles.

I think where I'm getting lost in ^ example is, for Child, it is not a leaf node, so we don't draw it's border in the new leaf-only GridRow code...so it needs to be drawn by `wrapCard` right? E.g. `wrapCard(Child)` would add both the parent padding/border and child's own padding/border? ...and it does that b/c when calling `wrapCard(Child)`, the child itself is already considered an open card?

Reply:
Yes, in that instance `parent` and `child` have their padding and border-left and border-right drawn by `wrapCard`, because the `parent` and `child` are both currently opened cards.

Reply:
Maybe another name for the function would help?

Reply:
Gotcha. Maybe `addNonLeafPaddingAndBorders`? :-) `addNotLeafWrappers`... Or just keeping it as `wrapCard` and having a jsdoc that mentions something like "adds non-leaf padding & borders, e.g. if the current row is a non-leaf, it will already be in open cards"...